src/main.py

```diff
@@ -0,0 +1,62 @@
+# main
+# -*- coding: utf-8 -*-
+import logging
+import asyncio
+from threading import Thread
+from aiogram import Bot, Dispatcher, types
+from aiogram.contrib.middlewares.logging import LoggingMiddleware
+from aiogram.utils import executor
+from fastapi import FastAPI
+import config
+import uvicorn
+from message_handler import message_handle
+from healthcheck import router as healthcheck_router
+
+
+# Initialize logging
+logging.basicConfig(level=config.LOG_LEVEL)
+logger = logging.getLogger(__name__)
+
+# Initialize FastAPI
+REST_API = FastAPI()
```

Comment:
UPPERCASE indicates constant in Python, which is quite confusing here :thinking: 

```suggestion
api = FastAPI()
```